File: apps/server_api/schema.py
from enum import Enum
from typing import Dict, List
from pydantic import BaseModel
class DeepWebInfo(BaseModel):
    keyword: str
    url: str

class DeepWebSuggest(BaseModel):
    url: str

class PushKeyword(BaseModel):
    keyword: str

class ServerHealth(BaseModel):
    cpu_percent: float
    memory_percent: float

File: apps/antd_deepweb_dashboard/src/schemas/schema.ts
export interface DeepWebInfo {
    keyword: string;
    url: string;
}

export interface DeepWebSuggest {
    url: string;
}

export interface PushKeyword {
    keyword: string;
}

export interface ServerHealth {
    cpu_percent: number;
    memory_percent: number;
}